akka-http-core/src/main/scala/akka/http/Http.scala

```diff
@@ -99,27 +99,34 @@ class HttpExt(config: Config)(implicit system: ActorSystem) extends akka.actor.E
     bindAndHandle(Flow[HttpRequest].mapAsync(handler), interface, port, backlog, options, settings, log)
 
   /**
-   * Transforms a given HTTP-level server [[Flow]] into a lower-level TCP transport flow.
+   * The type of the server-side HTTP layer as a stand-alone BidiStage
+   * that can be put atop the TCP layer to form an HTTP server.
+   *
+   * {{{
+   *                +------+
+   * HttpResponse ~>|      |~> ByteString
```

Comment:
This convention already appears to be established.
The scaladoc to `BidiFlow::atop`, for example, says,:

> By convention protocol stacks are growing to the left: the right most is the bottom layer, the closest to the metal.